plugin/cancel/cancel.py

```diff
@@ -1,20 +1,32 @@
 # to disable command cancellation, comment out this entire file.
-# you may also wish to adjust the commands in misc/cancel.talon.
+# you may also wish to adjust the commands in plugin/cancel/cancel.talon.
 from talon import actions, speech_system
 
-# To change the phrase used to cancel commands, you must also adjust misc/cancel.talon
-cancel_phrase = "cancel cancel".split()
+# To change the phrase used to cancel commands, you must also adjust plugin/cancel/cancel.talon
+cancel_phrases = ["cancel cancel"]
+
+# Alternatively, if you want to use a two-word phrase that
+# can include homophones, you can use this code instead:
+# cancel_words = ["cancel", "counsel", "council"]
+# cancel_phrases = [
+#     " ".join([word1, word2])
+#     for word1 in cancel_words
+#     for word2 in cancel_words
+# ]
 
 
 def pre_phrase(d):
-    n = len(cancel_phrase)
     if "text" in d and "parsed" in d:
-        before, after = d["text"][:-n], d["text"][-n:]
-        if after != cancel_phrase:
-            return
-        # cancel the command
-        d["parsed"]._sequence = []
-        actions.app.notify(f"Command canceled: {' '.join(before)!r}")
+        text_str = " ".join(d["text"])
+        for phrase in cancel_phrases:
+            if text_str.endswith(phrase):
```

Comment:
This now matches words that end in the first word of the phrase. This won't be a problem for `cancel` since I can't come up with any English words that end in `cancel`, but could very much be a problem for any synonyms that anyone adds to the list.